### ISSUE TITLE
Bug: In Preferences, have to click OK twice to exit Preferences dialog

### ISSUE THREAD

**Issue by mpogue2** [mpogue2/SquareDesk#564]
This bug is NOT present in 0.9.4, so it's something done since then.  Possibly the upgrade to Qt5.15?  Not sure.

Same is true for clicking on Cancel.  Have to do it twice.
However, clicking on red X in upper left corner instantly closes the dialog.

**Comment by mpogue2** [comment]
With the 0.9.5 source code built against the Qt 5.15 release, it fails.
With the 0.9.5 source code built against the Qt 5.9.3 release, it works as expected.
So, this is definitely something associated with the latest Qt release.

**Comment by mpogue2** [comment]
Import/Export dialogs do NOT have this problem -- they work as expected.  So, it's something that we are doing to make the PreferencesDialog special.

**Comment by mpogue2** [comment]
Pressing RETURN or ESC both work correctly (equiv to OK and Cancel respectively).
Clicking on something like a checkbox and then on Cancel or OK does not work (have to click on Cancel or OK a second time).

**Comment by mpogue2** [comment]
I removed all the code between creating the PreferencesDialog and the exec(), and it WORKS.
Narrowed it down to this one line of code:
    prefsManager.populatePreferencesDialog(prefDialog);
If that is commented out, then the dialog no longer requires two clicks to exit.

**Comment by mpogue2** [comment]
OK, so in the process of testing by removing lines of code, I accidentally turned all my preferences to zero in /Library/Preferences/com.zenstarstudio.SquareDesk.plist .  So, black for colors, no path to songs, etc.  So, I needed to start over here.  

I moved the com.zenstarstudio.SquareDesk.plist file out of the Preferences folder.  I knew that SquareDesk would make a new one on next startup.  I then reset all the Preferences, and guess what?  The problem is now gone.  Every attempt to close the Preferences dialog with either OK or Cancel happens on the first click.
Attached is the bad one: [com.zenstarstudio.SquareDesk.plist.zip](https://github.com/mpogue2/SquareDesk/files/4750158/com.zenstarstudio.SquareDesk.plist.zip)

Further exploration showed that it stops working whenever the font size is larger than the default. (!!).  I usually set the font size to bigger than default, so when I did, the problem was there.  Command-0 (go to default font size) makes the problem go away in the Preferences dialog.  

Should be able to narrow it down further by commenting code out in the font-size changing function.  

How weird.
